Only in protobuf-2.5.0-patched/: dummy
diff -c -r protobuf-2.5.0/src/google/protobuf/stubs/platform_macros.h protobuf-2.5.0-patched/src/google/protobuf/stubs/platform_macros.h
*** protobuf-2.5.0/src/google/protobuf/stubs/platform_macros.h	2013-02-26 17:56:38.000000000 +0000
--- protobuf-2.5.0-patched/src/google/protobuf/stubs/platform_macros.h	2021-01-17 12:28:46.000000000 +0000
***************
*** 46,51 ****
--- 46,54 ----
  #elif defined(__QNX__)
  #define GOOGLE_PROTOBUF_ARCH_ARM_QNX 1
  #define GOOGLE_PROTOBUF_ARCH_32_BIT 1
+ #elif defined(__AARCH64EL__)
+ #define GOOGLE_PROTOBUF_ARCH_ARM 1
+ #define GOOGLE_PROTOBUF_ARCH_64_BIT 1
  #elif defined(__ARMEL__)
  #define GOOGLE_PROTOBUF_ARCH_ARM 1
  #define GOOGLE_PROTOBUF_ARCH_32_BIT 1
